src/NuGet.Core/NuGet.Packaging/PublicAPI/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -1 +1 @@
-NuGet.ContentModel.ContentItemCollection.PopulateItemGroups(NuGet.ContentModel.PatternSet definition, System.Collections.Generic.IList<NuGet.ContentModel.ContentItemGroup> contentItemGroupList) -> void
```

Comment:
Unnecessary line ending changes that can be reverted.
Probably a consequence of the cleanup.